doc/source/development/policies.rst

```diff
@@ -0,0 +1,51 @@
+.. _develop.policies:
+
+********
+Policies
+********
+
+.. _policies.version:
+
+Version Policy
+~~~~~~~~~~~~~~
+
+.. versionchanged:: 1.0.0
+
+Pandas uses a variant of `SemVer`_ to govern deprecations, API compatibility, and version numbering.
+
+A pandas release number is made up of ``MAJOR.MINOR.PATCH``.
+
+API breaking changes should only occur in **major** releases. Theses changes will be documented,
+with clear guidance on what is changing, why it's changing, and how to migrate existing code to the
+new behavior.
+
+Whenever possible, a deprecation path will be provided rather than an outright breaking change.
+
+Pandas will introduce deprecations in **minor** releases. These deprecations will
+preserve the existing behavior while emitting a warning that provide guidance
+
+* How to achieve similar behavior if an alternative is available
+* The pandas version in which the deprecation will be enforced.
+
+We will not introduce new deprecations in patch releases.
+
+Deprecations will only be enforced in **major** releases.
```

Comment:
This can maybe use a bit more explanation (I am not sure everybody reads "enforced" the same, I am myself sometimes confused about this terminolog). Attempt:

... This means that a deprecated behaviour which is introduced in a minor release (eg 1.2) will continue to work for other minor releases in that series (other 1.x releases), but raise a warning. The deprecation will then only be removed and the behaviour changed in the next major release.